### PR TITLE
Add other members for BeforeInstallPrompt event.

### PR DESCRIPTION
**Pull request by amusaber950**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/beforeinstallpromptevent/BeforeInstallPromptEvent/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: BeforeInstallPromptEvent.BeforeInstallPromptEvent()
```
Comment (teoli2003) [inline files/en-us/web/api/beforeinstallpromptevent/BeforeInstallPromptEvent/index.md:2]: This should be:
```suggestion
title: BeforeInstallPromptEvent()
```

File: files/en-us/web/api/beforeinstallpromptevent/BeforeInstallPromptEvent/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+tags:
+  - API
+  - Constructor
+  - Reference
+  - BeforeInstallPromptEvent
+browser-compat: api.BeforeInstallPromptEvent.BeforeInstallPromptEvent
+---
+{{DefaultAPISidebar("")}}
+
+The **`BeforeInstallPromptEvent()`** constructor creates a new {{domxref("BeforeInstallPromptEvent")}} object which is fired at the {{domxref("Window.onbeforeinstallprompt")}} handler before a user is prompted to "install" a web site to a home screen on mobile.<
```
Comment (teoli2003) [inline files/en-us/web/api/beforeinstallpromptevent/BeforeInstallPromptEvent/index.md:13]: Extraneous _<_
```suggestion
The **`BeforeInstallPromptEvent()`** constructor creates a new {{domxref("BeforeInstallPromptEvent")}} object which is fired at the {{domxref("Window.onbeforeinstallprompt")}} handler before a user is prompted to _install_ a web site to a home screen on mobile.
```

File: files/en-us/web/api/beforeinstallpromptevent/BeforeInstallPromptEvent/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Syntax
+
+```js
+new BeforeInstallPromptEvent(type)
+new BeforeInstallPromptEvent(type, int);
```
Comment (teoli2003) [inline files/en-us/web/api/beforeinstallpromptevent/BeforeInstallPromptEvent/index.md:19]: No final semi-colon:
```suggestion
new BeforeInstallPromptEvent(type, int)
```

File: files/en-us/web/api/beforeinstallpromptevent/BeforeInstallPromptEvent/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+### Parameters
+
+- `type`
+  - :
```
Comment (teoli2003) [inline files/en-us/web/api/beforeinstallpromptevent/BeforeInstallPromptEvent/index.md:25]: We are missing the description, as well the one of int.

File: files/en-us/web/api/beforeinstallpromptevent/BeforeInstallPromptEvent/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+## Examples
+
+Fill in a simple example that nicely shows a typical usage of the API, then perhaps some more complex examples (see our guide on how to add [code examples](/en-US/docs/MDN/Contribute/Structures/Code_examples) for more information).
```
Comment (teoli2003) [inline files/en-us/web/api/beforeinstallpromptevent/BeforeInstallPromptEvent/index.md:29]: This is the default block! We need an example.

File: files/en-us/web/api/beforeinstallpromptevent/platforms/index.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  - API
+  - Property
+  - Reference
+  - platforms
+  - BeforeInstallPromptEvent
+browser-compat: api.BeforeInstallPromptEvent.platforms
+---
+{{DefaultAPISidebar("")}}
+
+The **`platforms`** property of the {{domxref("BeforeInstallPromptEvent")}} interface
```
Comment (teoli2003) [inline files/en-us/web/api/beforeinstallpromptevent/platforms/index.md:14]: Missing description here!

This is a stub, and we need it to be documented or removed of the PR, before merging.

File: files/en-us/web/api/beforeinstallpromptevent/userChoice/index.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+tags:
+  - API
+  - Property
+  - Reference
+  - userChoice
+  - BeforeInstallPromptEvent
+browser-compat: api.BeforeInstallPromptEvent.userChoice
+---
+{{DefaultAPISidebar("")}}
+
```
Comment (teoli2003) [inline files/en-us/web/api/beforeinstallpromptevent/userChoice/index.md:13]: Same this page is a stub and we either want it to be filled, or removed from this PR.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thanks a lot!

The constructor page is not too far from our standard for landing, but the two members need more work.

**Comment by teoli2003** [comment]
@amusaber950 Do you think that you will be able to complete the missing part? If not, no worry, but I think I will close this PR then.

**Comment by sideshowbarker** [comment]
@teoli2003, should we close this one?

**Comment by teoli2003** [comment]
Closing for lack on response. We will happily consider a new PR with the missing parts.

**Pull request closed without merging**